internal/controller/helper.go

```diff
@@ -286,19 +280,15 @@ func handleIgnoreAnnotationPropagation(
 			log.V(1).Info("Patched Child's annotations for ignore operation", "ChildResource", childObj.GetName(), "Operation", opResult)
 		}
 		return nil
-	})
+	}); errs != nil {
+		errs = append(errs, err)
```

Comment:
If we use the error from `meta.EachListItem()` the `errs` slice isn't needed anymore.